pypit/core/arextract.py

```diff
@@ -239,15 +239,15 @@ def extract_boxcar(image,trace_in, radius_in, ycen = None):
     return fextract
 
 
-def extract_optimal(waveimg, imgminsky, ivar, mask, oprof, skyimg, rn_img, box_radius, specobj):
+def extract_optimal(waveimg, imgminsky, ivar, mask, oprof, skyimg, rn2_img, box_radius, specobj):
```

Comment:
I think you need to update the doc string too.

Reply:
Done.